app/assets/stylesheets/jobs.scss

```
@@ -1,4 +1,10 @@
 
 .w-80 {
-  min-width: 80%;
-}
\ No newline at end of file
+  width: 80%;
+}
+.appJobsCount {
```

```suggestion
}

.app-job-count {
```
Let's give it a space here and use hyphens in our class names. It's a pattern we follow in OOD, the hyphens anyway.